apps/frontend/src/lib/components/ChatInput.svelte

```diff
@@ -10,12 +10,13 @@
 	}
 
 	let {
-		value = $bindable(),
+		value = $bindable(''),
 		isStreaming = false,
 		onSubmit,
 		placeholder = 'Message...'
 	}: Props = $props();
 
+	let isEmpty = $derived(!String(value ?? '').trim());
```

Comment:
After the `$bindable('')` change above, are you sure `String()` is still necessary here? Mostly nitpicking. ;)